tests/trainer/test_lr_finder.py

```diff
@@ -155,7 +154,7 @@ def test_accumulation_and_early_stopping(tmpdir):
         'Learning rate was not altered after running learning rate finder'
     assert len(lrfinder.results['lr']) == 100, \
         'Early stopping for learning rate finder did not work'
-    assert lrfinder._total_batch_idx == 100 * 2, \
+    assert lrfinder._total_batch_idx == 190, \
```

Comment:
how is this related?

Reply:
not really but the test was skipped completely

Reply:
:(